_layouts/default.html

```diff
@@ -13,56 +13,86 @@
-                    {% assign contentsTitle = page.toc_title | default: 'Contents' %}
-                    {% include toc.html html=content class='menu-list' h_min=2 h_max=3 contents_title=contentsTitle %}
-                </div>
-                {% elsif page.menubar %}
-                <div class="column is-4-desktop is-4-tablet">
-                    {% include menubar.html %}
-                </div>
-                {% endif %}
-                <div class="column {{ content_width }}">
-                    {% include tabs.html %}
-                    {% include showcase.html %}
-                    {% include sponsors.html %}
-                    {% include gallery.html %}
-                    {% include twoColumnGallery.html %}
-                    {{ content }}
-                </div>
-                {% if site.posts and page.show_sidebar %}
-                <div class="column is-4-desktop is-4-tablet">
-                    {% include latest-posts.html %}
-                </div>
-                {% endif %}
+</div>
+{% include header.html %}
+
+{% include callouts.html %}
+<section class="section">
+    <div class="container">
+        <div class="columns is-multiline">
+            {% if page.menubar_toc %}
+            <div class="column is-4-desktop is-4-tablet">
+                {% assign contentsTitle = page.toc_title | default: 'Contents' %}
+                {% include toc.html html=content class='menu-list' h_min=2 h_max=3 contents_title=contentsTitle %}
+            </div>
+            {% elsif page.menubar %}
+            <div class="column is-4-desktop is-4-tablet">
+                {% include menubar.html %}
+            </div>
+            {% endif %}
+            <div class="column {{ content_width }}">
+                {% include tabs.html %}
+                {% include showcase.html %}
+                {% include sponsors.html %}
+                {% include gallery.html %}
+                {% include twoColumnGallery.html %}
+                {{ content }}
+            </div>
+            {% if site.posts and page.show_sidebar %}
+            <div class="column is-4-desktop is-4-tablet">
+                {% include latest-posts.html %}
             </div>
+            {% endif %}
         </div>
-    </section>
-    {% unless page.hide_footer %}
-        {% include footer.html %}
-    {% endunless %}
-    <script src="{{ site.baseurl }}/assets/js/app.js" type="text/javascript"></script>
-    {%- include footer-scripts.html -%}
+    </div>
+</section>
+{% unless page.hide_footer %}
+{% include footer.html %}
+{% endunless %}
+<script src="{{ site.baseurl }}/assets/js/app.js" type="text/javascript"></script>
+{%- include footer-scripts.html -%}
 
-    {% unless page.hide_goto_top %}
-        {% include goto_top.html %}
-    {% endunless %}
+{% unless page.hide_goto_top %}
+{% include goto_top.html %}
+{% endunless %}
+
+<script>
+    document.addEventListener("DOMContentLoaded", function() {
```

Comment:
This is likely something that can be done in a separate JS file.

Reply:
https://github.com/FRCTeam1987/teambroncobots.com/commit/5a553cf06c247558777edc0fe740c1cec0e9c854